### PR TITLE
dbeaver/pro#2676 use fully qualified names checkbox

### PR DESCRIPTION
**Pull request by HocKu7**

### CODE REVIEW

File: plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/PostgreUtils.java [hunk 1]
```diff
@@ -616,13 +616,14 @@ public static void setArrayParameter(JDBCPreparedStatement dbStat, int index, Li
         }
     }
 
-    public static String getViewDDL(DBRProgressMonitor monitor, PostgreViewBase view, String definition) throws DBException {
+    public static String getViewDDL(DBRProgressMonitor monitor, PostgreViewBase view, String definition,
+                                    Map<String, Object> options) throws DBException {
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/PostgreUtils.java]:
```suggestion
    public static String getViewDDL(
        DBRProgressMonitor monitor,
        PostgreViewBase view,
        String definition,
        Map<String, Object> options
    ) throws DBException {
```
You might also want to specify nullability annotations here.

File: plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/PostgreViewBase.java [hunk 2]
```diff
@@ -155,18 +140,28 @@ public String getObjectDefinitionText(DBRProgressMonitor monitor, Map<String, Ob
         return ddl.toString();
     }
 
+    private void fetchViewQueryResult(JDBCSession session) throws SQLException, DBException {
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/PostgreViewBase.java:144]: It's used in just a single place. Why extract it?

File: plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/struct/DBStructUtils.java [hunk 3]
```diff
@@ -130,7 +130,11 @@ public static String getTableDDL(@NotNull DBRProgressMonitor monitor, @NotNull D
         return generateTableDDL(monitor, table, options, addComments);
     }
 
-    public static <T extends DBSEntity> void generateTableListDDL(@NotNull DBRProgressMonitor monitor, @NotNull StringBuilder sql, @NotNull Collection<T> tablesOrViews, Map<String, Object> options, boolean addComments) throws DBException {
+    public static <T extends DBSEntity> void generateTableListDDL(@NotNull DBRProgressMonitor monitor,
+                                                                  @NotNull StringBuilder sql,
+                                                                  @NotNull Collection<T> tablesOrViews,
+                                                                  Map<String, Object> options,
+                                                                  boolean addComments) throws DBException {
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/struct/DBStructUtils.java]: Same as in the first comment

File: plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/PostgreViewBase.java (hunk 2 above)
Comment (HocKu7) [reply plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/PostgreViewBase.java:144]: By my opinion it easy to read the host function

Comment (ShadelessFox) [reply plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/PostgreViewBase.java:144]: Debatable

**Review by E1izabeth — APPROVED** [review]

File: plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/PostgreViewBase.java [hunk 4]
```diff
@@ -48,72 +48,57 @@
...
-    {
+    public PostgreViewBase(PostgreSchema catalog) {
         super(catalog);
     }
 
     public PostgreViewBase(
         PostgreSchema catalog,
-        ResultSet dbResult)
-    {
+        ResultSet dbResult) {
```
Comment (E1izabeth) [inline plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/PostgreViewBase.java]:
```suggestion
        ResultSet dbResult
    ) {
```
Formatting was fine here

**Review by ShadelessFox — APPROVED** [review]

**Review by E1izabeth — APPROVED** [review]

**Comment by DariaMarkaryan** [comment]
Verified

**Pull request merged**